src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/17.diff.sql

```diff
@@ -0,0 +1,31 @@
+﻿/****************************************************************************************
+Guidelines to create migration scripts - https://github.com/microsoft/healthcare-shared-components/tree/master/src/Microsoft.Health.SqlServer/SqlSchemaScriptsGuidelines.md
+This diff is broken up into several sections:
+ - The first transaction contains changes to tables and stored procedures.
+ - The second transaction contains updates to indexes.
+ - IMPORTANT: Avoid rebuiling indexes inside the transaction, it locks the table during the transaction.
+******************************************************************************************/
+
+SET XACT_ABORT ON
+
+BEGIN TRANSACTION
+GO
+
+-- Data Type INT = 56
+IF EXISTS (SELECT * FROM sys.sequences WHERE Name = 'WorkitemKeySequence' AND system_type_id = 56)
+    BEGIN
+
+        DECLARE @sql NVARCHAR(MAX);
+        DECLARE @newWorkitemKeyStartVal INT;
+
+        -- Daily average is at ~50. Hence we buffer 100 on top of the last workitem key sequence
+        SET @newWorkitemKeyStartVal = (NEXT VALUE FOR dbo.WorkitemKeySequence) + 100;
+
+        SET @sql = CONCAT(N'DROP SEQUENCE dbo.WorkitemKeySequence; CREATE SEQUENCE dbo.WorkitemKeySequence AS BIGINT START WITH ', STR(@newWorkitemKeyStartVal), 'INCREMENT BY 1 MINVALUE ' + STR(@newWorkitemKeyStartVal) + ' NO CYCLE CACHE 10000');
```

Comment:
Is the dynamic sql execution will also be inside the transaction? Or should we have a separate transaction inside the dynamic sql. What is the drop succeeded and the create failed?

Reply:
outer transaction should still take care of this scenario. if needed, we can add an inner transaction part of dynamic sql.